back/engines/commercial/bulk_import_ideas/app/jobs/bulk_import_ideas/idea_pdf_import_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module BulkImportIdeas
+  class IdeaPdfImportJob < ApplicationJob
+    self.priority = 60
+
+    def run(files, import_user, locale, phase, personal_data_enabled)
```

Comment:
```suggestion
    def run(idea_import_files, import_current_user, locale, phase, personal_data_enabled)
```
Is `files` a list of `IdeaImportFile` instances? Would this name make sense then?